gluster-exporter/conf/global.conf.sample

```diff
@@ -0,0 +1,8 @@
+{
+	"gluster-mgmt": "glusterd2",
+	"gluster-work-dir": "/var/lib/glusterd2/",
+	"gluster-binary-path": "/usr/sbin/glusterd2",
+	"gd2-rest-endpoint": "http://192.168.10.11:24007",
```

Comment:
We also need gd2 auth/tls related settings. Which can be added later, just leaving a note here

Reply:
Hi, how can we enable TLS on exporter?